build-support/bin/cherry_pick.sh

```diff
@@ -73,6 +74,9 @@ for MILESTONE in $MILESTONES; do
   BRANCH_NAME="cherry-pick-$PR_NUM-to-$MILESTONE"
   git checkout -b "$BRANCH_NAME" FETCH_HEAD
   if git cherry-pick "$COMMIT"; then
+    if [[ -n $REMOTE ]]; then
+      git push -u "$REMOTE" "$BRANCH_NAME"
+    fi
```

Comment:
Since the GitHub Action doesn't get to participate in prompting, we need to push the branch.

Reply:
I assume the GitHub cli won't even try to prompt in this case, it'll detect the branch already exists on a remote?